### DISCUSSION TITLE
Configure Time Zone

### DISCUSSION THREAD

**Discussion by ryanjohnsontv** [tteck/Proxmox#784] [Ideas]
I recently made the move from a bare metal Home Assistant OS configuration to a virtualized instance across multiple LXCs. I noticed my Node RED container, created using the latest install script, was firing time-based automations 6 hours earlier than expected. Long story short, running `dpkg-reconfigure tzdata` and choosing my time zone in the Node RED container and rebooting was a quick and easy fix. I was curious if either flagging certain containers that are affected by UTC vs local time zone would be a welcome addition in the documentation, or if baking time zone selection into the install process would be a feature of interest.

Regardless, thank you for your time and energy creating and updating this repository. It's made my experience moving fully to Proxmox a very smooth one!

**Comment by dmshimself** [comment]
Yes I find I typically change the TZ, change the Locale and regenerate, add a regular user, add that user as a sudo-er, add ssh-copy-id certs for the user too.
